plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableHandle.java

```diff
@@ -56,6 +57,7 @@ public IcebergTableHandle(
             @JsonProperty("tableName") String tableName,
             @JsonProperty("tableType") TableType tableType,
             @JsonProperty("snapshotId") Optional<Long> snapshotId,
+            @JsonProperty("schemaJson") String schemaJson,
```

Comment:
schemaJson -> icebergSchema ?

We already have `schemaName` field in this class which makes this new field rather ambiguous.

Reply:
then `tabelSchemaJson` (adding "iceberg" generally doesn't mean anything within iceberg connector)